src/handlers/notify.rs

```diff
@@ -110,7 +107,7 @@ async fn process_publish_jobs(
     jobs: Vec<PublishJob>,
     client: Arc<relay_client::websocket::Client>,
     response: &mut Response,
-    request_id: uuid::Uuid,
+    uuid: uuid::Uuid,
```

Comment:
Up to you - personally find `request_id` more mnemonic

Reply:
`uuid` also has collisions with the package name